java8/src/main/java/io/cucumber/java8/DataTableDefinitionBody.java

```diff
@@ -0,0 +1,8 @@
+package io.cucumber.java8;
+
+import java.util.Map;
+
+@FunctionalInterface
+public interface DataTableDefinitionBody<T> {
```

Comment:
This would be a `DataTableEntryDefinitionBody`. Not to be confused with the `TableRowDefinitionBody`, `TableCellDefinitionBody` and actual `DataTableDefinitionBody`. 

See [`JavaDataTableTypeDefinition`](https://github.com/cucumber/cucumber-jvm/blob/master/java/src/main/java/io/cucumber/java/JavaDataTableTypeDefinition.java#L71) for the full set.